astropy/io/ascii/tests/test_ecsv.py

```diff
@@ -285,8 +285,17 @@ def assert_objects_equal(obj1, obj2, attrs, compare_class=True):
 srd = SphericalRepresentation(sr, differentials=sd)
 sc = SkyCoord([1, 2], [3, 4], unit='deg,deg', frame='fk4',
               obstime='J1990.5')
-scc = sc.copy()
+scd = SkyCoord([1, 2], [3, 4], [5, 6], unit='deg,deg,m', frame='fk4',
+               obstime=['J1990.5'] * 2)
+scc = scd.copy()
```

Comment:
Total nitpick, but call this `scdc` or keep it with `sc`!

Reply:
Done.